code/debris/debris.cpp

```diff
@@ -475,7 +498,7 @@ object *debris_create(object *source_obj, int model_num, int submodel_num, vec3d
 				db->lifeleft = -1.0f;		// large hull pieces stay around forever
 			}
 		} else {
-			db->lifeleft = (myrand() * RAND_MAX_1f) * 2.0f + 0.1f;
+			db->lifeleft = (frand() * 2.0f + 0.1f) * sqrtf(radius);
```

Comment:
This means that larger debris will stick around longer, because otherwise it would only be 0.1-2 seconds, which is far too quick for large or medium sized debris. I found the square root of the radius to give the nicest results for sticking around appropriately long for its size, but still going away relatively quickly, and also maintained the 10m fighter -> 1m debris default values (i.e. it boils back down to the original expression for retail inputs)

Reply:
That is a good idea.

Reply:
I looked at everything else and I'm ready to approve it once you have that comment in.